app/models/knight.rb

```diff
@@ -1,2 +1,19 @@
 class Knight < Piece
+  def obstructed_diagonally?
```

Comment:
Yep, adding the `(*)` at the end of each `def obstructed_` line gives me 0 offenses too. I've saved those changes to **knight.rb**.